docs/element-template-templatet-print.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "Parkausweisdaten an Druckstraße übergeben",
+  "id": "de.muenchen.digiwf.druckstrasse.integration.DatenAnDruckstrasse",
+  "version": 1,
+  "description": "Templatexxx",
+  "appliesTo": [
+    "bpmn:CallActivity"
+  ],
+  "properties": [
+    {
+      "label": "Baustein",
+      "type": "String",
+      "editable": false,
+      "value": "GenericDruckBeauftragen",
+      "binding": {
+        "type": "property",
+        "name": "calledElement"
+      }
+    },
+    {
+      "label": "Event Topic",
+      "type": "String",
+      "value": "",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "target": "app_topic_name"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Event Message",
+      "type": "String",
+      "editable": false,
+      "value": "genericEvent",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "target": "app_message_name"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Type Header",
+      "type": "String",
+      "editable": false,
+      "value": "requestPrint",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "target": "app_type_name"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+	  "label": "Versandart",
+	  "description": "Versandart des Dokuments (POSTVERSAND, DIENSTVERSAND) eingeben.",
+      "type": "String",
+      "value": "",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "target": "BewohnerVersandart"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+	    {
+	  "label": "Abholort",
+	  "description": "Abholort des Druckdokuments nur bei Versandart DIENSTVERSAND eingeben.",
+      "type": "String",
+      "value": "",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "target": "AbholortDienstversand"
```

Comment:
=> pickupLocation?

I think the other two (druckDaten, parkausweisVorlage) might also be incorrect

Reply:
correct template uploaded